code/datums/wires/airlock.dm

```diff
@@ -192,6 +213,15 @@
 		if(WIRE_ZAP1, WIRE_ZAP2) // Ouch.
 			if(isliving(usr))
 				A.shock(usr, 50)
+		if(WIRE_UNRESTRICTED_EXIT) // If you cut this wire, the unrestricted helper goes way. If you mend it, it'll go "haywire" and pick a new direction at random. Might have to cut/mend a time or two to get the direction you want.
+			if(!A.unres_sensor) //only works if the "sensor" is installed (a variable that we assign to the door either upon creation of a door with unrestricted directions or if an unrestricted helper is added to a door in mapping)
+				return
+			if(!mend)
+				A.unres_sides = !A.unres_sides
+				A.update_appearance()
+			else
+				A.unres_sides = pick(NORTH, SOUTH, EAST, WEST)
```

Comment:
if unres_sides is a bit value why do an invert here rather than explicitly setting it to NONE or using like direction_reverse or whatever the helper is called